make.js

```diff
@@ -32,7 +32,7 @@ async.waterfall([
 
 function getSystemJsBundleConfig(cb) {
   let config = {
-    baseURL: '..',
+    baseURL: '.',
```

Comment:
and, again, this is intentional
if not do this hack
npm and webpack users will be doing
`import 'ng2-bootstrap/ng2-bootstrap'`
and system.js users will do
`import 'ng2-bootstrap'`
but same code should work for both platforms